RobotCode/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -53,15 +63,19 @@ private void configurePrimaryBindings()
 
 	private void configureSecondaryBindings()
 	{
-		Joystick secondaryController;
-		JoystickButton secondaryTrigger;
+		secondaryController = new Joystick(1);
+		JoystickButton rightTrigger = new JoystickButton(secondaryController,
+				LogitechControllerButtons.triggerRight);
+
+		rightTrigger.whileTrue(new ShooterCommand(ShooterSubsystem));
 	}
 
 	// Add autonomous options to the SendableChooser
 	private void addAutonomousOptions()
 	{
-		autoChooser = AutoBuilder.buildAutoChooser();
+		
 		ShuffleboardTab.add("Auto Chooser", autoChooser);
+		autoChooser = AutoBuilder.buildAutoChooser();
```

Comment:
This must be above `ShuffleboardTab.add`. That line uses the autoChooser, so it has to be instantiated prior to being used.